examples/extensions/remove_entire_sitelink_campaign_extension_setting.rb

```diff
@@ -0,0 +1,210 @@
+#!/usr/bin/env ruby
+# Encoding: utf-8
+#
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Removes the entire sitelink campaign extension setting by removing both the
+# sitelink campaign extension setting itself and its associated sitelink
+# extension feed items. This requires two steps, since removing the campaign
+# extension setting doesn't automatically remove its extension feed items.
+#
+# To make this example work with other types of extensions, find
+# `ExtensionType::SITELINK` and replace it with the extension type you wish to
+# remove.
+
+require 'optparse'
+require 'google/ads/google_ads'
+require 'date'
+
+def remove_entire_sitelink_campaign_extension_setting(customer_id, campaign_id)
+  # GoogleAdsClient will read a config file from
+  # ENV['HOME']/google_ads_config.rb when called without parameters
+  client = Google::Ads::GoogleAds::GoogleAdsClient.new
+
+  mutate_operations = []
+
+  # Creates a mutate operation that contains the campaign extension setting
+  # operation to remove the specified sitelink campaign extension setting.
+  mutate_operations << create_sitelink_campaign_extension_setting_mutate_operation(
+    client,
+    customer_id,
+    campaign_id,
+  )
+
+  # Gets all sitelink extension feed items of the specified campaign.
+  google_ads_service = client.service.google_ads
+  extension_feed_item_resource_names = get_all_sitelink_extension_feed_items(
+    client,
+    google_ads_service,
+    customer_id,
+    campaign_id,
+  )
+
+  # Creates mutate operations, each of which contains an extension feed item
+  # operation to remove the specified extension feed items.
+  mutate_operations += create_extension_feed_item_mutate_operations(
+    client,
+    extension_feed_item_resource_names)
```

Comment:
Nit: Add comma and put closing paren on its own line.

Reply:
Done.